src/components/Visualization/Visualization.js

```diff
@@ -178,27 +187,29 @@ export const Visualization = ({
 
     const mountAndObserveContainerRef = useCallback(
         (node) => {
+            console.log('resize container', node)
             if (node === null) {
                 return
             }
 
             containerRef.current = node
             sizeObserver.observe(node)
 
-            return sizeObserver.disconnect
+            return () => sizeObserver.unobserve(node)
```

Comment:
I'm not sure if this change actually makes a difference in practice because both elements probably unmount at the same time and calling `.disconnect()` multiple times is probably not a big deal. Having said that, I'd still be in favour of keeping it: in the functions we call observe on a single node, so in the cleanup we should stick to just unobserve that particular node.

Reply:
Yes, I left it there, but it didn't seem to make a difference.
What made the difference was to add the `requestAnimationFrame`.